captcha.module

```diff
@@ -327,13 +328,17 @@ function captcha_form_alter(array &$form, FormStateInterface $form_state, $form_
       '#open' => TRUE,
     ];
 
-    if ($captcha_point !== NULL && $captcha_point->getCaptchaType()) {
+    if (!empty($captcha_point) && $captcha_point->status()) {
       $markup = '<a href="@change">change</a>';
       $captcha_element['#title'] = $captcha_point->status() ? t('CAPTCHA: challenge "@type" enabled', ['@type' => $captcha_point->getCaptchaType()]) : t('CAPTCHA: challenge "@type" disabled', ['@type' => $captcha_point->getCaptchaType()]);
-      $captcha_point->status() ? $captcha_element['#description'] = t('Untrusted users will see a CAPTCHA here (<a href="@settings">general CAPTCHA settings</a>).',
-        ['@settings' => Url::fromRoute('captcha_settings')->toString()]) : $captcha_element['#description'] = t('CAPTCHA disabled, Untrusted users won\'t see the captcha (<a href="@settings">general CAPTCHA settings</a>).',
-        ['@settings' => Url::fromRoute('captcha_settings')->toString()]
-        );
+      if ($captcha_point->status()) {
```

Comment:
You never get into the else case anymore because you call status() already on the outer if.